plugins/lookup/tss.py

```diff
@@ -26,6 +26,14 @@
         description: The integer ID of the secret.
         required: true
         type: int
+    _fetch_attachments:
+        description: The boolean flag which indicate attached file will get downloaded or not.
+        required: false
+        type: bool
```

Comment:
Please add the collection version this option will be added to to both options:
```suggestion
        type: bool
        version_added: 7.0.0
```

Reply:
Added version_added